lib/iiif_print.rb

```diff
@@ -89,4 +90,43 @@ def iiif_print_config?
       end
     end
   end
+
+  # @api public
+  #
+  # Map the given model's metadata to the given IIIF version spec's metadata structure.  This
+  # is intended to be a drop-in replacement for `Hyrax::IiifManifestPresenter#manifest_metadata`.
+  #
+  # @param model [Object]
+  # @param version [Integer]
+  # @param fields [Array<IiifPrint::Metadata::Field>, Array<#name, #label>]
+  # @return [Array<Hash>]
+  #
+  # @see specs for expected output
+  #
+  # @see Hyrax::IiifManifestPresenter#manifest_metadata
+  def self.manifest_metadata_for(model:,
+                                 version: config.default_iiif_manifest_version,
+                                 fields: default_fields_for(model))
+    Metadata.manifest_for(model: model, version: version, fields: fields)
+  end
+
+  def self.metadata_fields
```

Comment:
We should have public documentation on what this looks like.  Alternatively we could remove this method and replace calls of `IiifPrint.metadata_fields` with `IiifPrint.config.metadata_fields`

Reply:
yeah that makes sense, now that we have that config, why not use it instead